app/flatpak-builtins-build-finish.c

```diff
@@ -235,7 +235,7 @@ collect_exports (GFile          *base,
     "share/icons",                        /* Icons */
     "share/dbus-1/services",              /* D-Bus service files */
     "share/gnome-shell/search-providers", /* Search providers */
-    "share/appdata",                      /* Copy appdata/metainfo files */
+    "share/metainfo",                     /* Copy appdata/metainfo files */
```

Comment:
Should we also be exporting `share/appdata`? That seems to be more commonly-included in apps on Flathub at the moment.

Or should we be taking `share/appdata` and `share/metainfo` as input, and merging them into `share/metainfo` in output?

Reply:
As far as I can tell flatpak-builder renames metainfo into appdata (hence my comment in 9541d4df58d146d2aad60e1c6bdef34f5c14bb57 ) : https://github.com/flatpak/flatpak-builder/blob/master/src/builder-manifest.c#L2519 , https://github.com/flatpak/flatpak-builder/issues/252 so just swapping `appdata` with `metainfo` is not sufficient.

I'm fine with using metainfo but that requires to either move files instead of just copying  or have flatpak-builder use a newer location.

Reply:
[Here's a PR](https://github.com/flatpak/flatpak-builder/pull/441) for making flatpak-builder use the new path. This PR will still need updating to check both paths so it works with released versions of flatpak-builder.